praw/models/reddit/subreddit.py

```diff
@@ -1393,6 +1405,19 @@ def update(
             "text_color": text_color,
             "text_editable": text_editable,
         }
+        if fetch:
+            _existing_data = [
+                template
+                for template in iter(self)
+                if template["id"] == template_id
+            ]
+            if len(_existing_data) == 0:
+                raise InvalidFlairTemplateID(template_id)
+            else:
+                existing_data = _existing_data[0]
+                for key, item in data.copy().items():
+                    if not bool(item):
+                        data[key] = existing_data[key]
```

Comment:
If you wanted, these three lines could be replaced with:

    data.update({key: value for key, value in existing_data.items() if value is not None})

Reply:
This is a much better solution than the code, thank you.